doc/tutorials/nvmx_virtualization.rst

```diff
@@ -0,0 +1,289 @@
+* The guest hypervisor, which we call L1 hypervisor, or L1 VMM
+* The nested guest, which we call L2 guests
+
+.. figure:: images/nested_1.png
+   :width: 700px
+   :align: center
+
+   Generic Nested Virtualization
+
+
+High Level ACRN Nested Virtualization Design
+********************************************
+
+The high-level design of nested virtualization is shown in :numref:`nested_virt_hld`.
+ACRN. Nested VMX is enabled by so by allowing a guest to use VMX
+instructions, and emulating them using the single level of VMX available
+in the hardware.
+
+A logical processor uses virtual-machine control data structures (VMCSs) while
+it is in VMX operation. The core of the NVMX emulation is ACRN builds VMCS02
+out of the VMCS12 which is built by L1 hypervisor, to actually run the L2 guest
+
+.. figure:: images/nested_arch_1.png
+   :width: 700px
+   :align: center
+   :name: nested_virt_hld
+
+   Nested Virtualization in ACRN
+
+
+Restrictions and constrains
+***************************
+
+Currently nested virtualization is considered as an experimental feature.
+As of today, the tested platforms are: Tiger Lake RVP and Kaby Lake NUC 
+(See :ref:`hardware`.)
+
+L1 VMs have the following restrictions:
+
+* KVM is the only L1 hypervisor supported by ACRN
+* KVM runs in 64-bit mode
+* KVM enables EPT for L2 guests
+* QEMU is used to launch L2 guests
+
+Constrains on L1 guest configuration:
+
+* Local APIC passthrough should be enabled
+* use SCHED_NOOP scheduler because ACRN can't receive timer interrupts
+  on LAPIC passthrough pCPUs
+
+L1 VM configuration
+*******************
+
+The nested virtualization feature is disabled by default in ACRN. You can
+enable it using the :ref:`Use the ACRN Configuration Editor <acrn_config_tool_ui>`
+with the following steps:
+
+- Configure system level features:
+
+   - Edit ``NVMX_ENABLED`` to ``y`` in ACRN scenario XML configuration to
+     enable nested virtualization
+
+   - Edit ``SCHEDULER`` to ``SCHED_NOOP``
+
+   .. code-block:: xml
+      :emphasize-lines: 3, 10
+
+      <FEATURES>
+         <RELOC>y</RELOC>
+         <SCHEDULER>SCHED_NOOP</SCHEDULER>
+         <MULTIBOOT2>y</MULTIBOOT2>
+         <ENFORCE_TURNOFF_AC>y</ENFORCE_TURNOFF_AC>
+         <RDT>
+            <RDT_ENABLED>n</RDT_ENABLED>
+            <CDP_ENABLED>n</CDP_ENABLED>
+         </RDT>
+         <NVMX_ENABLED>y</NVMX_ENABLED>
+         <HYPERV_ENABLED>y</HYPERV_ENABLED>
+         <IOMMU_ENFORCE_SNP>n</IOMMU_ENFORCE_SNP>
+         <ACPI_PARSE_ENABLED>y</ACPI_PARSE_ENABLED>
+         <L1D_VMENTRY_ENABLED>n</L1D_VMENTRY_ENABLED>
+         <MCE_ON_PSC_DISABLED>n</MCE_ON_PSC_DISABLED>
+         <IVSHMEM>
+            <IVSHMEM_ENABLED>n</IVSHMEM_ENABLED>
+            <IVSHMEM_REGION/>
+         </IVSHMEM>
+      </FEATURES>
+
+Currently the Service OS's virtual UART interrupt doesn't work with LAPIC
+passthrough, which may prevent Service OS from booting. If the nVMX enabled
```

Comment:
Is `nVMX` the same as `NVMX`? If so, I would recommend we use only one capitalization for it.